akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -64,22 +64,20 @@ class ClientServerSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll
   "The low-level HTTP infrastructure" should {
 
     "properly bind a server" in {
-      val (hostname, port) = SocketUtil.temporaryServerHostnameAndPort()
       val probe = TestSubscriber.manualProbe[Http.IncomingConnection]()
-      val binding = Http().bind(hostname, port).to(Sink.fromSubscriber(probe)).run()
+      val binding = Http().bind("0.0.0.0", 0).to(Sink.fromSubscriber(probe)).run()
```

Comment:
since `0.0.0.0` binds to all IP's, I guess `127.0.0.1` is a bit more conservative?
```suggestion
      val binding = Http().bind("127.0.0.1", 0).to(Sink.fromSubscriber(probe)).run()
```